yb-voyager/cmd/import.go

```diff
@@ -397,8 +397,16 @@ func registerFlagsForTarget(cmd *cobra.Command) {
 			"number of cores N and use N/4 as parallel jobs. "+
 			"Otherwise, it fall back to using twice the number of nodes in the cluster. "+
 			"Any value less than 1 reverts to the default calculation.")
-	BoolVar(cmd.Flags(), &tconf.EnableYBAdaptiveParallelism, "enable-adaptive-parallelism", true,
-		"Adapt parallelism based on the resource usage (CPU, memory) of the target YugabyteDB cluster.")
+	// BoolVar(cmd.Flags(), &tconf.EnableYBAdaptiveParallelism, "enable-adaptive-parallelism", true,
+	// 	"Adapt parallelism based on the resource usage (CPU, memory) of the target YugabyteDB cluster.")
+
+	cmd.Flags().Var(&tconf.AdaptiveParallelismMode, "adaptive-parallelism",
+		"The mode for adaptive parallelism behavior: disabled, balanced, aggressive (default balanced)"+
```

Comment:
The mode for adaptive parallelism on how to control parallelism based on resource usage (CPU, memory) of the target YugabyteDB cluster: disabled, balanced, aggressive (default balanced)

just mentioning what is adaptive parallelism which we had in previous flag